demo/java/src/test/java/io/github/cosinekitty/astronomy/demo/MainTests.java

```diff
@@ -4,14 +4,13 @@
 import org.junit.jupiter.api.Test;
 
 import java.time.Instant;
-import java.util.Date;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 public class MainTests {
     @Test
     void main() {
         String time = "2022-01-01T12:00:00.000Z";
-        assertEquals(time, new Time(Date.from(Instant.parse(time))).toString());
```

Comment:
I must not be running this test as part of my automatic test suite, because there is no longer a `Time` constructor that accepts a `Date` parameter. I would have noticed this sooner otherwise. I will fix this in another commit.